hnn_core/parallel_backends.py

```diff
@@ -58,6 +58,18 @@ def _gather_trial_data(sim_data, net, n_trials):
     return dpls
 
 
+def _get_mpi_env():
+    """Set some MPI environment variables."""
+    my_env = os.environ.copy()
+    if 'win' not in sys.platform:
+        my_env["OMPI_MCA_btl_base_warn_component_unused"] = '0'
+
+    if 'darwin' in sys.platform:
+        my_env["PMIX_MCA_gds"] = "^ds12"  # open-mpi/ompi/issues/7516
```

Comment:
Looks like these issues are dependent on the ompi version. Are there other issues that would necessitate **ompi>=ver** in the install instructions?

Reply:
I don't foresee any. We don't have users install openmpi directly, so are somewhat dependent on the versions that work best for each OS (installed by conda or Microsoft MPI).